docs/reference/mobile/android/local-deploy.md

```diff
@@ -2,78 +2,35 @@
 
 # Connect to a local deployment
 
-This page explains how to connect an Android client to a local deployment of SpatialOS. You can do this from inside your [Unity Editor](#in-editor), using the [Unity Remote](#unity-remote) app, using an [Android Emulator](#android-emulator) or using your own [Android device](#android-device).
-
 Before reading this document, make sure you are familiar with:
 
-* [Setting up Android Support for the GDK]({{urlRoot}}/reference/mobile/android/setup)
-* [Ways to test your Android client]({{urlRoot}}/reference/mobile/android/run-client)
-
-## Prepare your project to connect to a local deployment {#prepare}
-
-To connect your Android device to a local deployment, you need to prepare its configuration. Follow the steps below to ensure any local deployment you launch via your Unity Editor is correctly set up.
-
-1. Open your project in your Unity Editor.
-1. Navigate to **SpatialOS** > **GDK Tools configuration** to open the configuration window.
-1. In the **Runtime IP for local deployment** field, enter your local machine's IP address. (You can find how to do this on the [Lifehacker website](https://lifehacker.com/5833108/how-to-find-your-local-and-external-ip-address).)
-1. Select **Save** and close the window.
-
-**Note:** If you are using one of our [Starter Projects]({{urlRoot}}/reference/glossary#starter-project), you can skip the **Create a mobile connector script** section below, as you already have one in your project.
-
-<%(#Expandable title="Create a mobile connector script")%>
-
-If you [added the GDK]({{urlRoot}}/projects/myo/setup) to an existing Unity project rather than using a Starter Project, then you also need to create and add a MonoBehaviour script to your Android client-worker GameObject. To do this:
+* [Setting up Android support for the GDK]({{urlRoot}}/reference/mobile/android/setup)
+* [Ways to test your mobile client]({{urlRoot}}/reference/mobile/run-client)
 
-1. Create a MonoBehaviour script which inherits from the [`MobileWorkerConnector`]({{urlRoot}}/api/mobile/mobile-worker-connector) and include the functionality you want. You can base your implementation on [the one](https://github.com/spatialos/gdk-for-unity-blank-project/blob/master/workers/unity/Assets/Scripts/Workers/AndroidClientWorkerConnector.cs) in our Blank Starter Project.
-1. In your Unity Editor, add the MonoBehaviour script to your Android client-worker GameObject.
-1. In your Unity Editor, navigate to your Android client-worker GameObject and ensure the `ShouldConnectLocally` checkbox is checked in the script's drop-down window of the Inspector window.
+## Start a local deployment
 
-<%(/Expandable)%>
-
-## Unity Editor{#in-editor}
-
-1. In your Unity Editor, Select **SpatialOS** > **Local launch**.<br>
+Before connecting your Android client-worker you need to start a local deployment.<br/>
```

Comment:
```suggestion
You need to start a local deployment **before** connecting your Android client-worker.
 
```

maybe swap the order to highlight that local deployment must be started

Reply:
```suggestion
Before connecting your Android client-worker you need to start a local deployment.

```

Reply:
Please don't use br's when we can just new newlines 😛

Reply:
it's not a hard requirement. it's depending on the project. For the FPS project we actually do have proper retry-logic